### ISSUE TITLE
COM_COPIED_OVERRIDDEN_METHOD false positive

### ISSUE THREAD

**Issue by boris-petrov** [spotbugs/spotbugs#495]
I have something like:

```java
class Foo {
  private static final String[] STRINGS = new String[] { "" };
  protected String[] getStrings() {
    return STRINGS;
  }
}

class Bar extends Foo {
  private static final String[] STRINGS = new String[] { "Bar's string" };
  @Override
  protected String[] getStrings() {
    return STRINGS;
  }
}
```

Which gives a `COM_COPIED_OVERRIDDEN_METHOD` which is wrong.

**Comment by iloveeclipse** [comment]
Boris, this warning is generated by fb-contrib library, so please report it there.
See http://fb-contrib.sourceforge.net/bugdescriptions.html

**Comment by boris-petrov** [comment]
Of course, my bad! Thanks!
